crypto/poly1305/poly1305_base2_44.c

```diff
@@ -59,6 +66,11 @@ int poly1305_init(void *ctx, const unsigned char key[16])
     poly1305_internal *st = (poly1305_internal *)ctx;
     u64 r0, r1;
 
+    if (FIPS_mode()) {
+        FIPSerr(ERR_LIB_FIPS, FIPS_R_NON_FIPS_METHOD);
+        return 1;
```

Comment:
Is there a specific convention to return 1 on error here, or are we inventing it just now seeing how 0 is returned on success? In the latter case, it may be slightly safer to return -1.

Reply:
The other poly1305_XXX functions return 1 on error. Principle of least surprises applies here. Arbitrarily returning -1 is unexpected IMHO.